source/scenario/globalScenarioTriggers.vwf.yaml

```diff
@@ -138,6 +138,8 @@ properties:
           - readBlackboard:
             - stepThroughProgramHint
       actions:
+      - writeToBlackboard:
+        - disableFailOnIncompleteProgram
```

Comment:
I'm a bit concerned about this fix... among other things, it relies on the triggers firing in the correct order (which is why we implemented the trigger groups in the first place).  It also only solves this immediate problem.  As we add custom failures, those might have similar problems, and since this doesn't address the root cause of the problem it won't fix those issues.  And, of course, it runs the risk that the other failure won't get re-enabled (though I don't think that's the case).

The right way to fix this is to dig into the trigger group code, and figure out why the trigger group isn't preventing the lower priority failure from occurring.